### PR TITLE
Punctuation and wording

### PR DESCRIPTION
**Pull request by Alan-Stewart**
I've been making small changes to punctuation (adding periods [.] to some of the lists) and some wording changes as I come across them.  I'll keep doing this as I work my way through the document.  Please only accept the changes that you feel are appropriate.

### CODE REVIEW

File: terminology.asciidoc [hunk 1]
```diff
@@ -32,13 +32,13 @@ In the EV Charging landscape, different market roles can be identified.
...
-|eMSP  |e-Mobility Service Provider: gives EV drivers access to charging services.
+|CPO   |Charging Point Operator. Operates a network of Charge Points.
+|eMSP  |e-Mobility Service Provider. Gives EV drivers access to charging services.
 |Hub   |Can connect one or more CPOs to one or more eMSPs.
-|NAP   |National Access Point: Provides a national Database with all (public) Charging Locations, information can be sent and retrieved from the NAP (that makes it different from a typical NSP).
-|NSP   |Navigation Service Provider: Provides EV drivers with location information of Charge Points.
-|Roaming Hub |See: Hub
-|SCSP  |Smart Charging Service Provider: Provided Smart Charging service to other parties might use a lot of different inputs to calculate Smart Charging Profiles.
+|NAP   |National Access Point. Provides a national database with all (public) charging locations. Information can be sent and retrieved from the NAP. This makes it different from a typical NSP.
+|NSP   |Navigation Service Provider. Similar to an eMSP. Provides EV drivers with location information of Charge Points. Usually only interested in Location information.
```
Comment (RobertdeLeeuw-IHomer) [inline terminology.asciidoc:39]: I don't like the part: Similar to an eMSP. An NSP only provides location information, no other services. So in my view they are not similar.

File: version_information_endpoint.asciidoc [hunk 2]
```diff
@@ -215,4 +215,4 @@ In this case the `credentials` module is not defined twice as this module is the
 [source,json]
 ----
 include::examples/version_details_example2.json[]
-----
\ No newline at end of file
+----
```
Comment (RobertdeLeeuw-IHomer) [inline version_information_endpoint.asciidoc:218]: What changed here? Seems like the same line, watch out with adding/removing white spaces or other non printed characters.

File: terminology.asciidoc [hunk 3]
```diff
@@ -210,4 +210,4 @@ Parties also will need to oblige to local legislation.
 
 When a new version of OCPI is implemented, the data exchanged via the old version does not have to be available via the newer version of OCPI.
 Hence, the Version end-point will probably have different end-points per version. So when an object is stored with a URL that contains a version,
-it is NOT REQUIRED to be available at a URL with a different version number.
```
Comment (RobertdeLeeuw-IHomer) [inline terminology.asciidoc:213]: What changed here? Seems like the same line, watch out with adding/removing white spaces or other non printed characters

Comment (RobertdeLeeuw-IHomer) [reply terminology.asciidoc:213]: This is just a new line at annd of line added.

File: terminology.asciidoc (hunk 1 above)
Comment (Alan-Stewart) [reply terminology.asciidoc]: I took out that line and committed it.  Did it go into the Pull Request or do I need to make another Pull Request every time I need to make changes to my changes?

**Pull request merged**